src/main/java/io/github/sefiraat/networks/network/SupportedRecipes.java

```diff
@@ -53,20 +53,8 @@ public boolean testRecipe(@Nonnull ItemStack[] input, @Nonnull ItemStack[] recip
         return true;
     }
 
-    public static boolean allowedRecipe(@Nonnull SlimefunItemStack i) {
-        return allowedRecipe(i.getItemId());
-    }
-
-    public static boolean allowedRecipe(@Nonnull String s) {
-        return !isBackpack(s);
-    }
-
-    public static boolean isBackpack(@Nonnull String s) {
-        return s.matches("(.*)BACKPACK(.*)");
-    }
-
     public static boolean allowedRecipe(@Nonnull SlimefunItem i) {
```

Comment:
It won't let me do a suggestion commit, mind changing `i` to `item`. My old code at fault but while we're here!